Money docs (#98)
pyvec/docs.pyvec.org: operations/support-money.rst

Na příkladu je vidět hned několik výhod tohoto modelu:

- Sponzor jen pošle peníze a nemusí se dál o nic starat.
- Peníze lze přijmout najednou i v případě, že se potom vyplácí postupně.
- Jiřík není na sponzorovi závislý. Hospoda na Mýtince jedná s Pyvcem, sponzoři se mohou měnit.
- Sponzorů může být i více než jeden a Hospoda na Mýtince stále jedná pouze s Pyvcem.
- Kdyby byla ve hře např. nájemní smlouva, podepisuje ji Pyvec, ne sponzor.
- Jiřík má za sebou právní entitu, která je schopna přijímat nebo vydávat faktury a má transparentní účet.

.. note::

>    Pyvec může věci :ref:`proplácet <jak-proplatit>` jen oproti účetnímu dokladu, tzn. účtence nebo faktuře. Pokud vám sponzor pošle více peněz než nakonec bylo potřeba, nelze vám je jen tak vyplatit. Můžete je ale v budoucnu čerpat na další věci pomocí :ref:`grantů <granty>`.

Reviewer: ```suggestion
   Pyvec může věci :ref:`proplácet <jak-proplatit>` jen oproti účetnímu dokladu, tzn. účtence nebo faktuře. Pokud vám sponzor pošle více peněz, než nakonec bylo potřeba, nelze vám je jen tak vyplatit. Můžete je ale v budoucnu čerpat na další věci pomocí :ref:`grantů <granty>`.
```

.. _granty:

Granty
^^^^^^

> Pokud na svou věc nemáte sponzora, můžete požádat Pyvec o grant. Pyvec disponuje penězi, které na jeho účtě zbyly z nevyčerpaných sponzorských darů, z neurčených darů od náhodných lidí nebo firem a z přebytků hospodaření konference `PyCon CZ <https://cz.pycon.org/>`__.

Reviewer: Proč náhodných?

Author: Anglicismus. Myšleno jako "random", jakože prostě lidi, kteří Pyvci jen tak přispějou na účet a nic za to nechtějí, přitom nejsou nutně třeba ani členové (nebo jsou, ale není to nic jako členský příspěvek, je to dar). Asi by se dalo vyhodit "náhodných" a ta věta by měla stále stejný smysl. Co myslíš?

Reviewer: Vyhodit se to dá. Stejně tak se dají vyhodit i lidi a firmy a nechat jen neurčené dary.

Author: Stručnost. Na té budu asi muset pracovat celý život a jestli je reinkarnace, tak i ten další život.

Author: 4c6b03787e506db44bcd858d638e990a068eee0f

> Pokud na svou věc nemáte sponzora, můžete požádat Pyvec o grant. Pyvec disponuje penězi, které na jeho účtě zbyly z nevyčerpaných sponzorských darů, z neurčených darů od náhodných lidí nebo firem, a z přebytků hospodaření konference `PyCon CZ <https://cz.pycon.org/>`__.

Reviewer 2: ```suggestion
Pokud na svou věc nemáte sponzora, můžete požádat Pyvec o grant. Pyvec disponuje penězi, které na jeho účtě zbyly z nevyčerpaných sponzorských darů, z neurčených darů od náhodných lidí nebo firem a z přebytků hospodaření konference `PyCon CZ <https://cz.pycon.org/>`__.
```

Viz [Psaní čárky před spojkami a, i, ani](https://prirucka.ujc.cas.cz/?id=153) (1.1. Poměr slučovací).
Tohle mi tak dlouho opravoval v materiálech Miro, až jsem si navykl na to dávat pozor :)

   - Kdo jste?
   - Kolik potřebujete? (i kdyby to měl být jen odhad nebo rozmezí)
   - Na jakou věc to je?
   - Proč? (čemu tím prospějeme, proč je to dobrý nápad, jaký to bude mít dopad?)

#. :term:`Výbor` žádost zváží a odhlasuje, zda ji schválí nebo zamítne. Toto se děje pro všechny žádosti minimálně jednou za měsíc na schůzi výboru, ale v závislosti na vytíženosti členů výboru se to může zvládnout i rychleji.
#. Výbor vás seznámí s výsledkem hlasování. Pokud žádost zamítne, měl by udat důvod, proč se tak stalo.
#. Zaplatíte věc ze svého a účetní doklady si necháte proplatit Pyvcem podle :ref:`tohoto návodu na proplácení <jak-proplatit>`. Pokud by šlo o velkou částku na fakturu, domluvíte se s Pyvcem, aby ji zaplatil rovnou.

   .. note::
      Pyvec může věci :ref:`proplácet <jak-proplatit>` jen oproti účetnímu dokladu, tzn. účtence nebo faktuře. Pokud vám byl udělen grant 4000 Kč, je tím míněno "proplatíme účetní doklady do výše 4000 Kč".

> #. Napíšete článek na `komunitní blog <https://blog.python.cz/>`__ (repo `zde <https://github.com/pyvec/blog.python.cz>`__), kde sepíšete (případně vyfotíte) výslednou věc. Jak se to povedlo, komu to pomohlo, jakou měli radost, co se naučili, apod.

Reviewer: ```suggestion
#. Napíšete článek na `komunitní blog <https://blog.python.cz/>`__ (`repozitář <https://github.com/pyvec/blog.python.cz>`__), kde sepíšete (případně vyfotíte) výslednou věc. Jak se to povedlo, komu to pomohlo, jakou měli radost, co se naučili apod.
```

   Příspěvek na blogu je **podmínkou** jednorázových grantů. Články nám pomáhají získávat peníze od sponzorů a informují komunitu o tom, co děláme. Pokud použijete grant a článek nenapíšete, nedostanete další grant.

   .. note::

>       Nemusíte válčit se systémem, na kterém blog běží. Článek napište v čem chcete, třeba v `Google Docs <https://docs.google.com/>`__, a lidi, kteří se kolem blogu motají, vám to do něj rádi pomohou vpravit.

Reviewer: ```suggestion
      Nemusíte válčit se systémem, na kterém blog běží. Článek napište v čem chcete, třeba v `Google Docs <https://docs.google.com/>`__, a lidi, kteří se kolem blogu motají, vám ho do něj rádi pomohou vpravit.
```

Příklad:

   Zuzka by chtěla natisknout třička pro `PyLadies <https://pyladies.cz/>`__. Na tričkách nechce loga sponzorů, takže sponzory nehledá, požádá Pyvec o grant. Do `#money <https://pyvec.slack.com/messages/C9E81JFS5/>`__ napíše:

>    *@board Ahoj, jsem organizátorka PyLadies. Rádi bychom natiskli trička. Podle toho, jaký bude zájem, budeme potřebovat 10 až 30 000 Kč. Trička jsou jedinou odměnou dobrovolníkům, posilují soudržnost mezi organizátory i absolventkami kurzů a zároveň šíří povědomí o PyLadies, když v nich lidi chodí po světě.*

Reviewer: ```suggestion
   *@board Ahoj, jsem organizátorka PyLadies v Zeleném Údolí. Rádi bychom natiskli PyLadies trička. Podle toho, jaký bude zájem, budeme potřebovat 10 až 30 000 Kč. Trička jsou jedinou odměnou dobrovolníkům, posilují soudržnost mezi organizátory i absolventkami kurzů a zároveň šíří povědomí o PyLadies, když v nich lidi chodí po světě. Jde o trička pro všechna města, jsme domluvené s organizátorkami z Modrého Města i Žlutého Vrchu.*
```

>    Výbor o tomto zahlasuje a odepíše, že grant bude přidělen. Následně Zuzka zjistí od lidí počty a velikosti triček, zadá zakázku firmě Tričkománie, s.r.o. a nechá u ní vystavit fakturu na Pyvec. Nakonec je to za 26 000 Kč. Protože jde o velkou částku, kterou Zuzka platit ze svého nechce, fakturu zaplatí přímo Pyvec. Když už je dost triček mezi lidma, Zuzka udělá pár fotek na mobil a napíše pár vět na `blog <https://blog.python.cz/>`__. Pro zajímavost přihodí i dvě čísla, kolik triček se vytisklo a jak rychle se rozdaly.

Reviewer: ```suggestion
   Výbor o tomto zahlasuje a odepíše, že grant bude přidělen. Následně Zuzka zjistí od lidí počty a velikosti triček, zadá zakázku firmě Tričkománie, s.r.o. a nechá u ní vystavit fakturu na Pyvec. Nakonec je to za 26 000 Kč. Protože jde o velkou částku, kterou Zuzka platit ze svého nechce a/nebo nemůže, fakturu zaplatí přímo Pyvec. Když už je dost triček mezi lidmi, Zuzka udělá pár fotek na mobil a napíše pár vět na `blog <https://blog.python.cz/>`__. Pro zajímavost přihodí i dvě čísla, kolik triček se vytisklo a jak rychle se rozdaly.
```

.. _pausalni-granty:

Paušální granty
^^^^^^^^^^^^^^^

Granty mohou být :ref:`jednorázové <granty>` nebo paušální. Ty paušální se hodí, pokud má nějaká skupina během své dlouhodobé činnosti více drobných výdajů. Příklad:

   Lumír je hlavním organizátorem kurzů pro začátečníky v Zeleném Údolí. Kurzy se konají pravidelně každý půlrok a při jejich organizaci je potřeba tu koupit lístečky, tu prodlužovačku, tam zase malé občerstvení. Nejsou to velké výdaje, takže je Lumír platí sám, ale když se v prosinci podíval do prasátka, zjistil, že v součtu to vlastně bylo dost peněz. Rozhodne se tedy požádat Pyvec o paušální grant.

> Postup získání grantu je :ref:`stejný jako u jednorázových <granty>`, akorát s tím rozdílem, že nežádáte o jednorázové peníze, ale o pravidelnou částku na měsíc. Opět platí, že Pyvec může věci :ref:`proplácet <jak-proplatit>` jen oproti účetnímu dokladu, tzn. účtence nebo faktuře. Grant 200 Kč/měsíc, je tedy míněn "proplatíme účetní doklady do výše 200 Kč měsíčně". Po získání paušálního grantu **nemusíte psát článek na blog**. (Ale můžete!) Příklad:

Reviewer: ```suggestion
Postup získání grantu je :ref:`stejný jako u jednorázových <granty>`, jen s tím rozdílem, že nežádáte o jednorázové peníze, ale o pravidelnou částku na měsíc. Opět platí, že Pyvec může věci :ref:`proplácet <jak-proplatit>` jen oproti účetnímu dokladu, tzn. účtence nebo faktuře. Grant 200 Kč/měsíc, je tedy míněn "proplatíme účetní doklady do výše 200 Kč měsíčně". Po získání paušálního grantu **nemusíte psát článek na blog**. (Ale můžete!) Příklad:
```

   Lumír do `#money <https://pyvec.slack.com/messages/C9E81JFS5/>`__ napíše:

>    *@board Zdar jak sviňa, v Zeleném Údolí děláme kurzy pro začátečníky, dva semestry ročně. Máme u toho malé výdaje, v podstatě kancelářské potřeby. Kurzy prošlo za poslední rok 40 lidí ročně a mají dlouhodobě skvělé ohlasy od účastníků. Pomohlo by nám proplatit 100 Kč měsíčně.*

Reviewer: \<sarkasmus\>Nechceš tam dát rovnou třeba *Čuš pičus?*\</sarkasmus\>

Author: Už jsi druhý (Pražák), kterého to zaujalo 😄 Zvláštní! Mě to přijde jako úplně normální (brněnský?) pozdrav 🤔

Author: Teď mám strašnou chuť někam přidat "čauky mňauky".

- netransparentní účet u `PayPal <https://www.paypal.com>`_

PayPal máme především kvůli prodeji lístků a vyplácení *Financial Aid* při organizaci konference `PyCon CZ <https://cz.pycon.org>`_, ale řešíme přes něj i jiné mezinárodní platby, např. sponzorské dary od `PSF <https://www.python.org/psf/>`_. Vše ostatní je na transparentním účtu.

Pyvec k Fio účtu disponuje debetní kartou, ale zatím se jejímu použítí spíše bráníme. Pokud jsou někde potřeba např. pravidelné platby kartou (např. při vedení účtu na `meetup.com <https://www.meetup.com/>`_), je z hlediska zabezpečení lepší, když tam dá člověk svoji kartu a peníze si poté nechá :ref:`proplatit <jak-proplatit>` standardním způsobem.

Hospodaření Pyvce
-----------------

Aktuální stav hospodaření Pyvce lze zjistit z každoročních účetních uzávěrek (přehledná tabulka kolik kde čeho proteklo) nebo dotazem e-mailem na :term:`účetní <Účetní>`, která by měla průběžně tušit. :term:`Pokladník` by taky měl průběžně tušit.

Naše :term:`účetní <Účetní>` nemá jiné API než e-mail, takže nelze stav hospodaření zjišťovat v reálném čase a např. jej transparentně zobrazovat na webu.

> Účetnictví Pyvce eviduje každou z podporovaných aktivit jako samostatné účetní středisko, což odborný termín pro pomyslnou "pojmenovanou obálku na peníze". Obálky umožňují rozlišit kolik peněz proteklo např. přes organizaci `PyCon CZ <https://cz.pycon.org>`_ nebo přes `PyLadies <https://pyladies.cz/>`_. Tyto obálky a jejich stav nijak zásadně neovlivňují způsob jakým udělujeme granty (tzn. nevadí, pokud je obálka v mínusu).

Reviewer 2: ```suggestion
Účetnictví Pyvce eviduje každou z podporovaných aktivit jako samostatné účetní středisko, což je odborný termín pro pomyslnou "pojmenovanou obálku na peníze". Obálky umožňují rozlišit kolik peněz proteklo např. přes organizaci `PyCon CZ <https://cz.pycon.org>`_ nebo přes `PyLadies <https://pyladies.cz/>`_. Tyto obálky a jejich stav nijak zásadně neovlivňují způsob jakým udělujeme granty (tzn. nevadí, pokud je obálka v mínusu).
```

> Účetnictví Pyvce eviduje každou z podporovaných aktivit jako samostatné účetní středisko, což je odborný termín pro pomyslnou "pojmenovanou obálku na peníze". Obálky umožňují rozlišit kolik peněz proteklo např. přes organizaci `PyCon CZ <https://cz.pycon.org>`_ nebo přes `PyLadies <https://pyladies.cz/>`_. Tyto obálky a jejich stav nijak zásadně neovlivňují způsob jakým udělujeme granty (tzn. nevadí, pokud je obálka v mínusu).

Reviewer: ```suggestion
Účetnictví Pyvce eviduje každou z podporovaných aktivit jako samostatné účetní středisko, což je odborný termín pro pomyslnou "pojmenovanou obálku na peníze". Obálky umožňují rozlišit, kolik peněz proteklo např. přes organizaci `PyCon CZ <https://cz.pycon.org>`_ nebo přes `PyLadies <https://pyladies.cz/>`_. Tyto obálky a jejich stav nijak zásadně neovlivňují způsob jakým udělujeme granty (tzn. nevadí, pokud je obálka v mínusu).
```

Na příkladu je vidět hned několik výhod tohoto modelu:

- Sponzor jen pošle peníze a nemusí se dál o nic starat.
- Peníze lze přijmout najednou i v případě, že se potom vyplácí postupně.
- Jiřík není na sponzorovi závislý. Hospoda na Mýtince jedná s Pyvcem, sponzoři se mohou měnit.
- Sponzorů může být i více než jeden a Hospoda na Mýtince stále jedná pouze s Pyvcem.
- Kdyby byla ve hře např. nájemní smlouva, podepisuje ji Pyvec, ne sponzor.
- Jiřík má za sebou právní entitu, která je schopna přijímat nebo vydávat faktury a má transparentní účet.

.. note::
   Pyvec může věci :ref:`proplácet <jak-proplatit>` jen oproti účetnímu dokladu, tzn. účtence nebo faktuře. Pokud vám sponzor pošle více peněz, než nakonec bylo potřeba, nelze vám je jen tak vyplatit. Můžete je ale v budoucnu čerpat na další věci pomocí :ref:`grantů <granty>`.

.. _granty:

Granty
^^^^^^

> Pokud na svou věc nemáte sponzora, můžete požádat Pyvec o grant. Pyvec disponuje penězi, které na jeho účtě zbyly z nevyčerpaných sponzorských darů, z neurčených darů a z přebytků hospodaření konference `PyCon CZ <https://cz.pycon.org/>`__. Slovo grant může někoho děsit, ale v našem podání to není nic byrokratického. Grant můžete žádat na libovolnou záležitost, která souvisí s misí Pyvce (viz stanovy: :ref:`ucel-spolku` a :ref:`formy-cinnosti-spolku`), ať už jde o stovky nebo desetitisíce korun: nájmy, trička, samolepky, projektor, ...

Reviewer: ```suggestion
Pokud na svou věc nemáte sponzora, můžete požádat Pyvec o grant. Pyvec disponuje penězi, které na jeho účtě zbyly z nevyčerpaných sponzorských darů, z neurčených darů a z přebytků hospodaření konference `PyCon CZ <https://cz.pycon.org/>`__. Slovo grant může někoho děsit, ale v našem podání to není nic byrokratického. Grant můžete žádat na libovolnou záležitost, která souvisí s misí Pyvce (viz stanovy: :ref:`ucel-spolku` a :ref:`formy-cinnosti-spolku`), ať už jde o stovky nebo desetitisíce korun: nájmy, trička, samolepky, projektor...
```
